api/import_export/export.py

```diff
@@ -166,6 +188,20 @@ def export_identities(organisation_id: int) -> typing.List[dict]:
     return [*identities, *traits]
 
 
+def export_edge_identities(organisation_id: int) -> typing.List[dict]:
+    identities = []
+    traits = []
+    identity_overrides = []
+    for environment in Environment.objects.filter(
+        project__organisation__id=organisation_id, project__enable_dynamo_db=True
+    ):
+        result = export_edge_identity_and_overrides(environment.api_key)
+        identities.extend(result[0])
+        traits.extend(result[1])
+        identity_overrides.extend(result[2])
```

Comment:
This is probably just preference, but I much prefer something like the following: 

```suggestion
        exported_identities, exported_traits, exported_overrides = export_edge_identity_and_overrides(environment.api_key)
        identities.extend(exported_identities)
        ...
```

Reply:
Done